cmd/pebble/cmd_run.go

```diff
@@ -31,7 +31,7 @@ import (
 	"github.com/canonical/pebble/internal/systemd"
 )
 
-var shortRunHelp = "Run the pebble environment"
+var shortRunHelp = "Run the pebble server"
```

Comment:
The reason it was worded as "pebble environment" is to give a feeling that you can have many of those. I was trying to avoid the somewhat usual feeling that there's that one server that you run for your whole machine, and instead give it a more user-level oriented taste: "You can have N environments, and this command runs one of them.

Does that make sense? Is there a way we can improve on the original wording while preserving that feeling? Maybe "plan" something something?

Reply:
Fair enough. We could say "a pebble server" rather than "the", but that doesn't sound quite right either. Reverted to "environment" for now.